docs/docs/guide-getting-started/simple-pub-sub.md

```diff
@@ -285,8 +295,15 @@ basis launch launch_two_process.yaml
 
 Two threads within one process (`launch_single_process.yaml`):
 ```yaml
-processes:
-  demo:
+recording:
+  directory: /tmp/
+  # list of glob expressions to match topics to
+  topics:
+    - /chatter
+    - /log
+groups:
+  simple_pub_sub:
+    process: False
```

Comment:
This is unnecessary

Reply:
to confirm: you mean the `process: False` bit?